### ISSUE TITLE
Will not work if sticky element sticks to bottom

### ISSUE THREAD

**Issue by naveedm1** [ryanwalters/sticky-events#12]
'STUCK' will not fire stuck event if sticky element  sticks to bottom. for example if you have a div with buttons and want to make them sticky while browser window is reduced change willfire unstuck will fire but STUCK will not , I guess it is not supported in your library.

**Comment by ryanwalters** [comment]
I agree this would be a good feature! I am open to PRs, otherwise I will see what it would take to implement this, possibly in the upcoming v3 release depending on complexity. Thanks for the suggestion.
